src/main/java/io/apimatic/core/types/BaseModel.java

```diff
@@ -148,9 +148,12 @@ private Method getMethodMatchingKey(String key, boolean getter) {
                 annotation = method.getAnnotation(JsonSetter.class);
             }
             if (annotation != null) {
-                if ((getter) && ((JsonGetter) annotation).value().equalsIgnoreCase(key)) {
+                String methodName = key;
+                if ((getter) && ((JsonGetter) annotation).value()
```

Comment:
Nits, Non-blocker: Extra round brackets can be removed on the `getter` conditional arg.

Reply:
Removed extra brackets